### PR TITLE
Python BDD tests in CI

### PR DESCRIPTION
**Pull request by Tatarinho**

### CODE REVIEW

**Review by alicja-gruzdz — APPROVED** [review]
Well done @Tatarinho

File: packages/host/src/lib/csi-controller.ts [hunk 1]
```diff
@@ -220,6 +220,14 @@ export class CSIController extends TypedEmitter<Events> {
         this.instancePromise = instanceMain();
     }
 
+    async cleanup() {
+        await Promise.all([
+            this.instanceAdapter.cleanup(),
+            this.communicationHandler.cleanup()
+        ]);
+        this.logger.error("Cleanup done...");
```
Comment (patuwwy) [inline packages/host/src/lib/csi-controller.ts:228]:
```suggestion
        this.logger.info("Cleanup done.");
```

Comment (Tatarinho) [reply packages/host/src/lib/csi-controller.ts:228]: This part will be removed from PR and added as separated fix in other place.

**Review by alicja-gruzdz — APPROVED** [review]

**Comment by alicja-gruzdz** [comment]
[bdd_python_ouput.txt](https://github.com/scramjetorg/transform-hub/files/8233780/bdd_python_ouput.txt)

**Pull request merged**